base_classes/nyaml/NXdata.yaml

```diff
@@ -237,6 +237,21 @@ NXdata(NXobject):
       .. note::  Attributes potentially containing multiple values
          (axes and _indices) are to be written as string or integer arrays,
          to avoid string parsing in reading applications.
+  \@AXISNAME_depends:
+    doc: |
+      Points to the path to a field defining the axis on which the ``AXISNAME`` axis depends.
+
+      This concept allows to store a link to an axis to a respective field the Nexus hierarchy
```

Comment:
```suggestion
      This concept allows to link an axis to a respective field in the Nexus hierarchy, thereby defining the physical quantity it represents.
```

Reply:
Added in new commit (so that nyaml and nxdl have the same text).